### PR TITLE
Included scripts to trigger and monitor pulse projects

### PR DESCRIPTION
**Pull request by bhuvnesh2703**
Included scripts to trigger and monitor pulse projects. The jobs which needs to trigger and monitor pulse projects should execute its task on the worker tagged as gpdb5-pulse-worker

### CODE REVIEW

File: concourse/pulse/api/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# vi: set ft=ruby :
```
Comment (ashwinstar) [inline concourse/pulse/api/Vagrantfile]: Do we really need this file ? Lets get only the files needed to trigger and monitor pulse.

File: concourse/pulse/api/Readme.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Pulse for Checkman
```
Comment (ashwinstar) [inline concourse/pulse/api/Readme.md]: README doesn't seem relevant.

**Comment by cjcjameson** [comment]
Pulse and Concourse are sibling ci systems -- they should both be under a top-level directory, such at `ci`. See my comments on #1406

**Comment by Chibin** [comment]
> Pulse and Concourse are sibling ci systems -- they should both be under a top-level directory, such at ci. See my comments on #1406

Done. Thanks

**Pull request merged**